src/Input/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import { Icon, SecondaryColor } from '../../index';
+import { Toggle, ToggleBall, ToggleSwitchWrapper, ToggleSwitchProps } from './';
+
+const ToggleSwitch: React.FunctionComponent<ToggleSwitchProps> = ({
+  defaultActive,
+  activeIconName,
+  inactiveIconName,
+  activeBackgroundColor,
+  inactiveBackgroundColor,
+  activeBallColor,
+  inactiveBallColor,
+  onChange,
+}) => {
+  const [active, setActive] = React.useState(defaultActive || false);
+
+  return (
+    <ToggleSwitchWrapper>
+      <Toggle
```

Comment:
Set a default className `aries-toggle-switch` and pass the className prop to `<Toggle>`. We usually set this on the root element. In this case your root element is the `<ToggleSwitchWrapper>` but that seems like it can be removed so you can pass it to `<Toggle>` instead.

Example: 
```
className={classNames('aries-toggle-switch', className)}
```
Reference: https://github.com/glints-dev/glints-aries/blob/v4/src/Input/Switch/Switch.tsx#L40